script/custom-eslint/enzyme-always-unmount/package.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "eslint-plugin-enzyme-always-unmount",
+        "version": "0.0.1",
```

Comment:
I don't know if versioning really matters on this since we're not publishing it but eslint requires that plugins are imported as modules.

Reply:
Like you said, this probably doesn't matter too much, but I'd still like to see it start at 1.0.0 as a best practice.

Reply:
Mostly because I'm just curious: @erikphansen Why 1.0.0 to start?